ntbs-service/Pages/Patients/Edit.cshtml

```diff
@@ -3,179 +3,218 @@
+                    <div class="govuk-radios__conditional govuk-radios__conditional--hidden"
+                        id="conditional-postcode-conditional">
+                        @{var postcodeErrorClass = !Model.IsValid("Patient.Postcode") ? errorClassName : ""; }
+                        <div class="nhsuk-form-group @postcodeErrorClass"
+                            v-bind:class="{ 'nhsuk-form-group--error': hasError}" aria-describedby="postcode-error">
+                            <span class="nhsuk-error-message" id="postcode-error">
+                                <span ref="errorField" asp-validation-for="Patient.Postcode"></span>
+                            </span>
+                            <input v-on:blur="validate" class="nhsuk-input nhsuk-input--width-10"
+                                asp-for="Patient.Postcode" type="text">
                         </div>
-                        <validate-input model="Patient" property="Postcode" inline-template>
-                            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-postcode-conditional">
-                                @{var postcodeErrorClass = !Model.IsValid("Patient.Postcode") ? errorClassName : ""; }
-                                <div class="nhsuk-form-group @postcodeErrorClass" v-bind:class="{ 'nhsuk-form-group--error': hasError}" aria-describedby="postcode-error">
-                                    <span class="nhsuk-error-message" id="postcode-error">
-                                        <span ref="errorField" asp-validation-for="Patient.Postcode"></span>
-                                    </span>
-                                    <input v-on:blur="validate" class="nhsuk-input nhsuk-input--width-10" asp-for="Patient.Postcode" type="text">
-                                </div>
-                            </div>
-                        </validate-input>
-                        <div class="nhsuk-radios__item">
-                            <input asp-for="Patient.NoFixedAbode" class="nhsuk-radios__input" type="radio" value="true">
-                            <label class="nhsuk-label nhsuk-radios__label" asp-for="Patient.NoFixedAbode">
-                                No fixed abode
-                            </label>
-                        </div>   
                     </div>
-                </fieldset>
+                </validate-input>
+                <div class="nhsuk-radios__item">
+                    <input asp-for="Patient.NoFixedAbode" class="nhsuk-radios__input" type="radio" value="true">
+                    <label class="nhsuk-label nhsuk-radios__label" asp-for="Patient.NoFixedAbode">
+                        No fixed abode
+                    </label>
+                </div>
             </div>
-            <br/>
+            <br />
+    </div>
+    <div>
+        <nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
```

Comment:
As we'll include this bit in all pages, could we make use of partial pages somehow so we don't have to repeat this? I don't know exactly how to use partial pages yet but I feel like they're supposed to be used for this kind of thing ha